crates/workspace/src/notifications.rs

```diff
@@ -551,8 +551,7 @@ where
             if let Err(err) = self.await {
                 log::error!("{err:?}");
                 if let Ok(prompt) = cx.update(|cx| {
-                    let detail = f(&err, cx)
-                        .unwrap_or_else(|| format!("{err:?}. Please try again.", err = err));
+                    let detail = f(&err, cx).unwrap_or_else(|| format!("{err}. Please try again."));
```

Comment:
Just to note, there's a compromise that will show all `.context`/`.with_context` entries with

```suggestion
                    let detail = f(&err, cx).unwrap_or_else(|| format!("{err:#}. Please try again."));
```

but up to you

Reply:
I think the context would be too noisy for a user-visible error message. This was the instigating reason to remove the backtrace:

![cleanshot_2024-06-05_at_10 52 14_2x_720](https://github.com/zed-industries/zed/assets/94272/3dc88342-37f9-4410-bfc4-ebbb6e8b9e17)


It should say "Call failed\n\nRPC request Call failed: failed to ring user" (though it would be better if we explained why we failed to ring the user)